3rdparty/downward/CMakeLists.txt

```diff
@@ -5,18 +5,21 @@ find_package(catkin REQUIRED)
 
 catkin_package()
 
-add_custom_command(
-  OUTPUT installed
-  COMMAND \$\(MAKE\) -f ${PROJECT_SOURCE_DIR}/Makefile
```

Comment:
You mean, we should add following lines?

```cmake
execute_process(COMMAND
  git config --global http.sslVerify false)
```

I use tarball, so I think we dont need to add this code?